interp/functions.go

```diff
@@ -261,6 +261,15 @@ func (p *interp) callBuiltin(op Token, argExprs []Expr) (value, error) {
 	}
 }
 
+// executes code using configured system shell
+func (p *interp) execShell(code string) *exec.Cmd {
+	shellInterpreter := p.shellCommand[0]
+	shellArgs := p.shellCommand[1:]
```

Comment:
Naming nit: how about we name these `executable` and `args`? We don't need the `shell` prefix in this context as this function already includes that in the name.